.npmignore

```diff
@@ -0,0 +1 @@
+# not lib
```

Comment:
Did you keep this here intentionally?

Reply:
Yes. In order to gitignore a directory but not ignore it from the published npm module, you must create an `.npmignore` file so that the `.gitignore` file isn't used when publishing